src/rt/sched/backpressure.h

```diff
@@ -0,0 +1,249 @@
+// Copyright Microsoft and Project Verona Contributors.
+// SPDX-License-Identifier: MIT
+#pragma once
+
+/**
+ * ## Introduction
+ *
+ * The primary goal of backpressure is to prevent runaway growth of a cown's
+ * message queue that might eventually result in a Verona application running
+ * out of memory. Cowns that receive messages at a higher rate than they are
+ * able to process are considered "overloaded" and may lead to this unbounded
+ * queue growth. Once an overloaded cown is identified, cowns sending messages
+ * to it will be temporarily descheduled ("muted") so that the overloaded cown
+ * can work through its queue. Similarly, cowns sending to muted receivers will
+ * also be muted so that cowns do not themselves become overloaded as a
+ * consequence of being muted. Muted cowns will be rescheduled after the
+ * receiver that triggered their muting (the "mutor") is no longer muted or
+ * overloaded.
+ *
+ * ## Detecting Overloaded Cowns
+ *
+ * An overloaded cown is one that experiences message queue growth as it is
+ * processing messages. Since it would be prohibitively expensive to calculate
+ * the length of a message queue directly, a count of messages removed from the
```

Comment:
I think you need to explain how "count of message removed from the queue" over time is different than counting the number of messages in the queue (in terms of performance).

This currently assumes that is obvious to the reader.